packages/lab/src/form-field-next/FormFieldNext.tsx

```diff
@@ -91,13 +95,16 @@ export const FormField = forwardRef(
       onFocus,
       readOnly = false,
       id: idProp,
+      validationStatus: validationStatusProp,
       ...restProps
     }: FormFieldProps,
     ref: ForwardedRef<HTMLDivElement>
   ) => {
     const labelId = useId();
     const helperTextId = useId();
 
+    const validationStatus = readOnly || disabled ? undefined : validationStatusProp;
```

Comment:
Changing the prop to an enum of `disabled | readonly | warning | error` would make this cleaner.

Reply:
cc @bhoppers2008 assuming disabled and readonly would take precedence over the statuses here, if we are to have separate props as is

Reply:
after conversation with @alycrys, decision stands as using guidance to say that using validation plus a disabled or readonly prop would be wrong. it is possible, but discouraged and doing so would make for broken behaviour which we won't be reponsible for
cc @joshwooding

Reply:
I don't think we should add conditions like this in the code. I believe these should be part of guidance in the documentation. 
I believe we should never hide the error status.

Reply:
It's an interesting point. I feel like I'm between the two decisions with good points on either side.

Even though it's going to be recommended that you do not use validation plus disabled/readonly I'm not sure whether it's worth deviating from the norm and restricting it. You technically could still have an API which was more aligned with HTML but have the disabled/readonly states always override the styles of the validation states

Reply:
> You technically could still have an API which was more aligned with HTML but have the disabled/readonly states always override the styles of the validation states

could you explain more what this would look like?